tools/quickstart/quickstart/cli.py

```diff
@@ -1,52 +1,146 @@
+import functools
 from textwrap import fill
 
 import click
+import pkg_resources
 
 from quickstart import bridge, docker, monitor, netstats, validator_account
-from quickstart.constants import NETSTATS_SERVER_LAIKA_BASE_URL
+
+DEFAULT_CONFIGS = ["Laika"]
+LAIKA, = DEFAULT_CONFIGS
+
+LAIKA_DOCKER_COMPOSE_FILE = "laika-docker-compose.yaml"
+LAIKA_BRIDGE_CONFIG_FILE = "laika-bridge-config.toml"
+LAIKA_NETSTATS_SERVER_BASE_URL = "https://laikanetstats.trustlines.foundation/"
+
+
+def config_file_getter(filename):
+    return functools.partial(
+        pkg_resources.resource_filename, __name__, f"configs/{filename}"
+    )
+
+
+class DefaultPathType(click.Path):
+    def __init__(self, default_path_lookups, *args, **kwargs):
+        """
+        Like the click path type, but additionally accepts a mapping for predefined paths
+        :param default_path_lookups: mapping from template name to lookup function
+        :param args: args for click.Path
+        :param kwargs: kwargs for click.Path
+        """
+        super().__init__(*args, **kwargs)
+
+        self.default_path_lookups = default_path_lookups
+
+    def convert(self, value, param, ctx):
+        if value in self.default_path_lookups:
+            value = self.default_path_lookups[value]()
+
+        return super().convert(value, param, ctx)
 
 
 @click.command()
+@click.option(
+    "--config",
+    "-c",
+    help="Config to use. [Default: Laika]",
+    type=click.Choice(DEFAULT_CONFIGS),
+    default=None,
+)
 @click.option(
     "--host-base-dir",
     help=(
-        "absolute path to use for docker volumes (only relevant when run from inside a docker "
+        "Absolute path to use for docker volumes (only relevant when run from inside a docker "
         "container itself)"
     ),
     type=click.Path(),  # Can not check for exist, because path is on host
     default=None,
 )
 @click.option(
     "--docker-compose-file",
-    help="path to the docker compose file to use",
-    type=click.Path(exists=True, dir_okay=False),
-    default="docker-compose.yaml",
+    help="Path to the docker compose file to use, or laika for the default laika docker compose file.",
```

Comment:
```suggestion
    help=f"Path to the docker compose file to use, or '{LAIKA}' for the default laika docker compose file.",
```

Reply:
I stumbled other this while reading. Even having seen the code above before. So I think adding these quotes make it more obvious that it is a specific keyword.
Moreover I would like to avoid stupid mistakes and replace the literal `laika` with `{LAIKA}`. Because that's what used to determine the keyword.
Same goes for the following `--bridge-config` option.

Reply:
done